pkg/qemu-ga/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM alpine:3.5
```

Comment:
yes, `tools/alpine` basically has a `apk` mirror of the union of all packages we build in `linuxkit`. You can create a multi-stage build from it and install just the packages you want. `pkg/dhcpcd/Dockerfile` has a good(?) example on how to use it. It installs a Alpine base system plus `dhcpcd`. Using the alpine base image allows us to create images in a more reproducible way as there are no network dependencies.

Since the qemu-ga is not in the base, you'd first have to add it to `./tools/alpine/packages`, build/push a new base image and then use it for your package.

This stuff has move a lot since